common/src/test/java/io/druid/common/utils/ServletResourceUtilsTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.common.utils;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ServletResourceUtilsTest
+{
+
+  @Test
+  public void testSanitizeException() throws Exception
+  {
+    final String message = "some message";
+    Assert.assertEquals(message, ServletResourceUtils.sanitizeException(new Throwable(message)).get("error"));
+    Assert.assertEquals("null", ServletResourceUtils.sanitizeException(null).get("error"));
+    Assert.assertEquals(message, ServletResourceUtils.sanitizeException(new Throwable()
```

Comment:
the intended priority in the face of null values is:

1. Throwable's message
1. Throwable.toString
1. "null"